### DISCUSSION TITLE
Order By Regex

### DISCUSSION THREAD

**Discussion by quickfiredigital** [ujjwalguptaofficial/JsStore#344] [Q&A]
I am search for multiple strings to appear in a table column using regex.
It would be great to have a way to order the results by the number of regex matches, is this possible?
Thanks

**Comment by ujjwalguptaofficial** [comment]
What do you mean by - order the results by no of regex matches ?. 

The order is done by columns - you can specify the columns and ordering type. 

Have a look at order - https://jsstore.net/docs/select/order-by#by-multiple-column

**Reply by veddishant** [reply]
i think @quickfiredigital is saying that using the regex(fuzzy search) fetching the results. but that are order by default or some column. but exact matched regex records comes first and partial match record come after onwards.

**Reply by ujjwalguptaofficial** [reply]
The where query can not be expected to return ordered results, for ordered results : it's better to rely on order query.
